src/index.js

```diff
@@ -25,6 +25,9 @@ console.log(`Reading ${legacyFilePath}.`);
 const legacyTranslations = {};
 parse(fs.readFileSync(legacyFilePath, "utf8"), {
   columns: ["key", "value"],
+    relax_column_count: true,
```

Comment:
I see you added this to solve an issue where column counts may not be identical. Will this cause the output to work as expected? I wish I had added tests for this code 😆